src/function/table.rs

```diff
@@ -313,7 +313,7 @@ fn hash_values(vs: &[Value]) -> u64 {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-struct Input {
+pub(crate) struct Input {
```

Comment:
I made this public in the crate so that I could access every call of every function efficiently.